README.md

```diff
@@ -234,8 +252,31 @@ JdPay::Service.notify_verify(xml_str, options = {})
   </jdpay>
   EOF
 
-  JdPay::Service.notify_verify(notyfy_xml)
+  JdPay::Service.notify_verify(xml_str)
+```
+
+### 支付成功页面跳转
+#### Name && Description
+> 用户支付成功后，京东支付系统根据支付接口中callbackUrl所传地址跳转到商户的支付成功页，并采用POST方式回传相应的订单参数。
+
+```ruby
+verify_redirection
 ```
+#### Definition
+```ruby
+JdPay::Service.verify_redirection(params, options = {})
+```
+#### Example
+```ruby
+# In your controller:
+begin
+  decrypted_params = JdPay::Service.verify_redirection(request.params, {})
+rescue => e
+  render :status => :bad_request
+  return
```

Comment:
```ruby
render status: :bad_request
```
1. [Use the Ruby 1.9 hash literal syntax when your hash keys are symbols](https://github.com/rubocop-hq/ruby-style-guide#hash-literals)

2. `return` is redundant.

Reply:
this way to early return from a rails controller. It's unnecessary at the end of a action.